src/main/java/cubicchunks/asm/mixin/core/common/MixinWorld.java

```diff
@@ -86,6 +88,21 @@
 	protected int minHeight = 0, maxHeight = 256;
 	private boolean wgenFullRelight;
 
+	@Override public void initCubicWorld() {
+		// Set the world height boundaries to their highest and lowest values respectively
+		ICubicChunksWorldType type = (ICubicChunksWorldType) this.getWorldType();
+		this.maxHeight = type.getDefaultMaxHeight();
+		this.minHeight = type.getDefaultMinHeight();
```

Comment:
If you use them as absolute height bounds - rename these methods to match.
When I proposed these names I had in mind using them as actual defaults (I thoiught it's obvious).